<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>League of Progra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Juego Nuevo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Juego</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>530</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>v 1.0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="campeon1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Campeon 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="campeon2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Campeon 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="campeon3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>220</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>Campeon 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>119</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Noob </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>119</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>119</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rage Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Empezar Partida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
